<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralGroupTab</class>
 <widget class="QWidget" name="GeneralGroupTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="logonNamePreWindows2000Label">
       <property name="text">
        <string>Group name (pre-Windows 2000):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="sam_name_domain_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sam_name_edit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="description_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="eMailLabel">
       <property name="text">
        <string>E-mail:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="email_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="notesLabel">
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="notes_edit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="groupScopeLabel">
       <property name="text">
        <string>Group Scope:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="groupTypeLabel">
       <property name="text">
        <string>Group Type:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="scope_combo"/>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="type_combo"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Display name: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string notr="true">PLACEHOLDER</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
